charts/flux2/tests/pre-install-job_test.yaml

```diff
@@ -4,7 +4,7 @@ templates:
 tests:
   - it: should have no annotations added by default
     asserts:
-      - isEmpty:
+      - notExists:
```

Comment:
This fails in CI with: `Assertion type `notExists` is invalid`

Reply:
Oh, then i use a different unit test plugin! I will change it!

Reply:
Or maybe there is a newer version and you could bump it in the GH workflow.

Reply:
switched back to `isEmpty`

I use for other helm chart testing this:

```
helm plugin install https://github.com/helm-unittest/helm-unittest
helm unittest charts/node-red
```

I will make an extra PR for this changes.